homeassistant/components/simplisafe/__init__.py

```diff
@@ -500,7 +500,7 @@ async def async_update(self):
 
         async def async_update_system(system):
             """Update a system."""
-            await system.update(cached=False)
+            await system.update(cached=system.version != 3)
```

Comment:
This is messy. Just return the update if the system version isn't 3.

Reply:
The reason why I did it this way is because the `await system.update()` call already existed before I added sensor support, and I thought there would be a reason for it.

What I changed vs the original is the argument `cached=False`.

If you don't think there's any reasons to return something on `async_update_system(system)` for SimpliSafe 2 systems, then yes, I can update it following your suggestion.

What do you think @bachya ?

Reply:
I see; thanks for that clarification. In that case, let's stick with what you have.